src/core/lib/iomgr/socket_utils_common_posix.cc

```diff
@@ -339,6 +339,10 @@ uint16_t grpc_htons(uint16_t hostshort) { return htons(hostshort); }
 
 uint16_t grpc_ntohs(uint16_t netshort) { return ntohs(netshort); }
 
+uint32_t grpc_htonl(uint32_t hostlong) { return htonl(hostlong); }
```

Comment:
should probably also add something for uv and windows

Reply:
At that point, honestly, I'd rather we implement our own version of these functions. They are simple enough that they don't warrant the portability hassle.

Reply:
given that we already have ntohs functions for all of them, I don't see the harm to adding ntohl functions too.